attributes/default.rb

```diff
@@ -67,6 +67,7 @@
                                                 intel-hpc-platform-core intel-hpc-platform-core-advisory intel-hpc-platform-hpc-cluster
                                                 intel-hpc-platform-compat-hpc intel-hpc-platform-core-intel-runtime]
 default['cluster']['intelhpc']['version'] = '2018.0-7'
+default['cluster']['intelhpc']['dependencies_dir'] = '/opt/intel/dependencies'
```

Comment:
I would not use a directory shared over NFS to cache this os packages. Shall we go for the existing `/opt/parallelcluster/sources/`?

Reply:
Done